### ISSUE TITLE
TensorFlow-dev-summit：那些TensorFlow上好玩的和黑科技

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/tensorflow#13]
# TensorFlow-dev-summit：那些TensorFlow上好玩的和黑科技 #

- Author: http://hacker.duanshishi.com/
- Origin: [http://hacker.duanshishi.com/?p=1780](http://hacker.duanshishi.com/?p=1780)
- Related:

> TensorBoard
> tf.summary.scalar(‘cross_entropy’, xent)
> tf.summary.scalar(‘accuracy’, accuracy)
> tf.summary.image(‘input’, x_image, 3)

> Hyperparameter Search

> Embedding Visualizer
